src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -949,6 +949,24 @@ trait BronzeTransforms extends SparkSessionWrapper {
 
   }
 
+  /**
+   *
+   * @param cloudProvider
+   * @param inputCol
+   * @param isMultiWorkSpaceDeployment
+   * @param organisationId
+   * @return
+   */
+  private def fetchClusterLogConfiguration(cloudProvider: String, inputCol: Column,
```

Comment:
what is inputCol? have some documentation regarding it.
add some documentation in the function.

Reply:
done

Reply:
please reformat the signature

Reply:
done